OracleSOASuite/dockerfiles/12.2.1.3/README.md

```diff
@@ -36,15 +36,15 @@ If that returns a username (which is the first field), you can skip the below us
 
 Once the `oracle` user is created, run the following commands as a root user:
 
-    # mkdir -p /u01/DockerVolume/SOAVolume/
-    # chown 1000:1000 /u01/DockerVolume/SOAVolume/
-    # chmod 700 /u01/DockerVolume/SOAVolume/
+    # mkdir -p /u01/DockerVolume/SOAVolume/SOA
+    # chown -R 1000:1000 /u01/DockerVolume/SOAVolume/
+    # chmod -R 700 /u01/DockerVolume/SOAVolume/
 
 # Database
 
 You need to have a running database container or a database running on any machine. The database connection details are required for creating SOA specific RCU schemas while configuring SOA domain. While using a 12.2.0.1 CDB/PDB DB, ensure PDB is used to load the schemas. RCU loading on CDB is not supported.
 
-Run the database container to host the RCU schemas using below steps:
+Run the database container to host the RCU schemas using below steps. For creating an Oracle Database container that uses data volumes to persist data, refer Oralce Enterprise Database documentation available at `https://container-registry.oracle.com`. 
```

Comment:
Typo in "Oracle"; also, you should refer to the documentation on GitHub as well, as some users may prefer to create their own image for Oracle Database using a more recent release than the one available on Container Registry.